packages/react-components/src/stories/Tag.stories.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { ComponentMeta } from '@storybook/react';
+import { Smiles } from '@livechat/design-system-icons/dist/material';
+
+import { Tag as TagComponent, ITagProps } from '../components/Tag';
+
+export default {
+  title: 'Components/Tag',
+  component: TagComponent,
+} as ComponentMeta<typeof TagComponent>;
+
+export const Tag = ({ children, ...args }: ITagProps): React.ReactElement => {
+  return (
+    <div className="spacer" style={{ display: 'flex' }}>
```

Comment:
AFAIR `spacer` class is helpful when you have many components under the div like:

```
<div className="spacer">
   <Button />
    <Button />
     <Button />
</div>
```

Do we need it here? Remember that it is going to be a part of "See code" panel.